nginx.conf

```diff
@@ -0,0 +1,39 @@
+user       root;  ## 
+worker_processes  1;  ## Default: 1
+#error_log  /usr/share/nginx/logs/error.log;
+#pid        /usr/share/nginx/logs/nginx.pid;
+worker_rlimit_nofile 8192;
+
+events {
+  worker_connections  1024;  ## Default: 1024
+}
+
+http {
+  index    index.html index.htm index.php;
+
+  default_type application/octet-stream;
+  log_format   main '$remote_addr - $remote_user [$time_local]  $status '
+    '"$request" $body_bytes_sent "$http_referer" '
+    '"$http_user_agent" "$http_x_forwarded_for"';
+  ##access_log   logs/access.log  main;
+  sendfile     on;
+  tcp_nopush   on;
+  server_names_hash_bucket_size 128; # this seems to be required for some vhosts
+
+
+  server { # simple reverse-proxy
+    listen       80;
+    server_name  _;
+
+    # serve static files
+    location /  {
+      root    /usr/share/nginx/html;
+      expires 30d;
+    }
+
+    location /api/ {
+      proxy_pass      http://asmodee-server:2620/;
```

Comment:
Is the host name here environment-specific?

Reply:
@takashiro 
If this docker container is started up using the [docker-compose script](https://github.com/Celthi/asmodee-werewolf-server/blob/dev/docker-compose.yaml) The host name then is not environment specific. Because using docker-compose it will asign the asmodee-server to the container that runs the server.
`Docker-compose up` will automate start the two container and assigns the host names.